composer.json

```diff
@@ -27,9 +27,18 @@
 	"config": {
 		"allow-plugins": {
 			"dealerdirect/phpcodesniffer-composer-installer": true
-		}
+		},
+		"platform-check": false
 	},
 	"require": {
-		"mlocati/ip-lib": "dev-master"
-	}
+		"mlocati/ip-lib": "dev-master",
+		"siddharth/wp-compat-checker": "dev-master",
```

Comment:
Do we want to move this library within the 10up org?

Reply:
@dkotter I discussed this with @jeffpaul, what we concluded is that if this proves viable then we can move it to 10up org.